azure-managed-lustre/connect-clients.md

```diff
@@ -26,13 +26,12 @@ Client machines running Linux can access Azure Managed Lustre. The basic client
 - **To achieve advertised performance**:
   - Clients must reside in the same Availability Zone in which the cluster resides.
   - [Enable accelerated networking](/azure/virtual-network/create-vm-accelerated-networking-cli#confirm-that-accelerated-networking-is-enabled) on all client VMs. If this option isn't enabled, then [fully enabling accelerated networking requires a stop/deallocate of each VM](/azure/virtual-network/accelerated-networking-overview#enabling-accelerated-networking-on-a-running-vm).
-- **Security type** - When selecting the security type for the VM, choose the Standard Security Type. Choosing Trusted Launch or Confidential types prevent the Lustre module from being properly installed on the client.
 
 The basic workflow is as follows:
 
 1. [Install or upgrade Lustre client software](#install-or-upgrade-lustre-client-software) on each client.
-1. Use the [`mount` command](#start-the-lustre-client-using-the-mount-command) to make the Azure Managed Lustre file system available on the client.
-1. When a client is no longer needed, use the `umount` command without the `-f` (force) or `-l` (lazy) options to cleanly unmount the client before shutting it down. Failure to properly unmount a client before rebooting or deprovisioning can result in performance issues for other clients.
+2. Use the [`mount` command](#start-the-lustre-client-using-the-mount-command) to make the Azure Managed Lustre file system available on the client.
+3. When a client is no longer needed, use the `umount` command without the `-f` (force) or `-l` (lazy) options to cleanly unmount the client before shutting it down. Failure to properly unmount a client before rebooting or deprovisioning can result in performance issues for other clients.
```

Comment:
```suggestion
1. When a client is no longer needed, use the `umount` command without the `-f` (force) or `-l` (lazy) options to cleanly unmount the client before shutting it down. Failure to properly unmount a client before rebooting or deprovisioning can result in performance issues for other clients.
```

Reply:
The flavor of Markdown we use follows this format for numbered lists (e.g., 1. 1. 1. for a numbered list renders as 1. 2. 3.). So if you can accept the suggestions to revert, that would be great (makes it easier to maintain if the list changes). Thanks!